### PR TITLE
DUMB MONEY - DUMB

### PR DESCRIPTION
**Pull request by Dumbmoneysol**
# Validate [DUMB](https://solscan.io/token/7AL9fsasEgYjoKKHbFGRYbrPHDX3sAywybvLuxmANogT)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/https://x.com/DUMBMONEYWINS/status/1803566308757581846?t=QShYHt5yTPyI6AJqozq0QA&s=19
- Coingecko/ CMC URL (If available):

## Validation (Please check off boxes):
- [X] The metadata provided in the PR matches what is on-chain (Mandatory)
- [X] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [X] My change matches the format in the file (no spaces between fields).
- [X] My token is already live and trading on Jupiter.
- [X] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [X] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Review by uponthethrees — COMMENTED** [review]
Mint address: 7AL9fsasEgYjoKKHbFGRYbrPHDX3sAywybvLuxmANogT

Symbol: DUMB

Token Decimals: 9

Token icon: https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f626c61636b2d63656e7472616c2d6c6f7573652d3538352e6d7970696e6174612e636c6f75642f697066732f516d516b784535674466735832554879345145454e7a6b426654374155735a3934364d736f47475939414d463572

On-chain metadata matches

No duplication on Jupiter's strict list

**Review by Dumbmoneysol — COMMENTED** [review]
Mint address: 7AL9fsasEgYjoKKHbFGRYbrPHDX3sAywybvLuxmANogT
 Symbol: DUMB
 Token Decimals: 9
 Token icon: https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f626c61636b2d63656e7472616c2d6c6f7573652d3538352e6d7970696e6174612e636c6f75642f697066732f516d516b784535674466735832554879345145454e7a6b426654374155735a3934364d736f47475939414d463572
 On-chain metadata matches
 No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -1794,3 +1794,4 @@ Mechelle Obema,OBEMA,CzkM8bzWFdXsjtZQnz2piTxJKPtJ5mfTL8S6sNZg7n7S,2,https://stat
 BILLY,BILLY,3B5wuUrMEi5yATD7on46hKfej3pfmd7t1RKgrsN3pump,6,https://cf-ipfs.com/ipfs/QmdRU5EkGt1VsgLsXJdHK5kjojQqggQHjhyqeJwmApR53R,true
 Genie,Genie,3Jjt8QhbqNoYfSQYHWf8ZsTJwE2CyvmUrzgzJD5Jpump,6,https://pbs.twimg.com/profile_images/1801011420525465600/2ywH00FN_400x400.jpg,true 
 Monkey Gold,MOGO,2WUecdL1GQA6pwQs7sMsCf2shEfZxWK2RaNHkthmLtmK,9,https://arweave.net/w_cYAcuwGOKxZWafjj-o14Ab_0PaaA_fZoQ8wqdy4rc,true
+DUMB MONEY,DUMB,7AL9fsasEgYjoKKHbFGRYbrPHDX3sAywybvLuxmANogT,9,https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f626c61636b2d63656e7472616c2d6c6f7573652d3538352e6d7970696e6174612e636c6f75642f697066732f516d516b784535674466735832554879345145454e7a6b426654374155735a3934364d736f47475939414d463572,true
```
Comment (damgit24) [inline validated-tokens.csv:1797]: Mint address: 7AL9fsasEgYjoKKHbFGRYbrPHDX3sAywybvLuxmANogT

Symbol: DUMB

Token Decimals: 9

Token icon: https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f626c61636b2d63656e7472616c2d6c6f7573652d3538352e6d7970696e6174612e636c6f75642f697066732f516d516b784535674466735832554879345145454e7a6b426654374155735a3934364d736f47475939414d463572

On-chain metadata matches

No duplication on Jupiter's strict list

Comment (damgit24) [inline validated-tokens.csv]: Mint address: 7AL9fsasEgYjoKKHbFGRYbrPHDX3sAywybvLuxmANogT

Symbol: DUMB

Token Decimals: 9

Token icon: https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f626c61636b2d63656e7472616c2d6c6f7573652d3538352e6d7970696e6174612e636c6f75642f697066732f516d516b784535674466735832554879345145454e7a6b426654374155735a3934364d736f47475939414d463572

On-chain metadata matches

No duplication on Jupiter's strict list

Comment (Dumbmoneysol) [reply validated-tokens.csv]: updated

**Review by damgit24 — COMMENTED** [review]
All updates correct.

Mint address: 7AL9fsasEgYjoKKHbFGRYbrPHDX3sAywybvLuxmANogT

Symbol: DUMB

Token Decimals: 9

Token icon: https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f626c61636b2d63656e7472616c2d6c6f7573652d3538352e6d7970696e6174612e636c6f75642f697066732f516d516b784535674466735832554879345145454e7a6b426654374155735a3934364d736f47475939414d463572

On-chain metadata matches

No duplication on Jupiter's strict list

**Review by uponthethrees — COMMENTED** [review]
Change are good.

**Comment by Jymfletzfur** [comment]
insufficient volume

**Pull request closed without merging**